src/db/supabase/types.ts

```diff
@@ -1,6 +1,31 @@
 export type Json = string | number | boolean | null | { [key: string]: Json | undefined } | Json[]
 
 export type Database = {
+  graphql_public: {
```

Comment:
so as not to interfere with future supabase:gen

Reply:
wdym? or is it just generated supabase types included into this commit?

Reply:
yes, just generated supabase types